cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -163,6 +163,49 @@ Typically, session management capabilities to track users after authentication m
 - Ensure entire cookie should be encrypted if sensitive data is persisted in the cookie
 - Define all cookies being used by the application, their name and why they are needed
 
+# HTML5 Web Storage API
+
+The Web Hypertext Application Technology Working Group (WHATWG) describes the HTML5 Web Storage APIs, localStorage and sessionStorage, as mechanisms for storing name-value pairs client-side.
+Unlike HTTP cookies, the contents of localStorage and sessionStorage are not automatically shared within requests or responses by the browser and are used for storing data client-side.
+
+## The localStorage API
+
+### Scope
+
+Data stored using the localStorage API is accessible by pages which are loaded from the same protocol, port and host.
```

Comment:
According to [this](https://html.spec.whatwg.org/multipage/webstorage.html#the-localstorage-attribute) + [this](https://html.spec.whatwg.org/multipage/origin.html#concept-origin), the origin that define access policy is defined by:
* A scheme (a scheme).
* A host (a host).
* A port (a port).
* A domain (null or a domain). Null unless stated otherwise. 

So it's important to mention the 4 elements explicitly.

Reply:
OK, thanks for the remark 😃 so forget my note.

Reply:
I don't mind at all to stay hand in hand with the official docs.

Reply:
Let's wait the feedback from the author.

Reply:
Happy to modify to use the scheme/host/port nomenclature.

Reply:
Please go ahead. Switching between the `host` and `domain` sometimes makes a difference with `same-site` and `same-origin`.

Reply:
I think I have resolved this. Please let me know if there are additional concerns...